test/main_test.cpp

```diff
@@ -0,0 +1,18 @@
+#include <gtest/gtest.h>
+
+#include "rclcpp/node.hpp"
+
+#define TEST_DOMAIN_ID 2
```

Comment:
Am besten hier mit constexpr, dann ist der Wert typisiert.
```suggestion
static constexpr size_t TEST_DOMAIN_ID 2
```

Reply:
Ist gemacht.